LICENSE

```diff
@@ -0,0 +1,202 @@
+                                 Apache License
```

Comment:
IIRC this license diverge from others we use. Which one is the proper one? Or, which one do we prefer to use?